charts/connect/main_test.go

```diff
@@ -0,0 +1,14 @@
+package connect
+
+import (
+	"log"
+	"os/exec"
+	"testing"
+)
+
+func TestHelmUnitTest(t *testing.T) {
+	out, err := exec.Command("helm", "unittest", ".").CombinedOutput()
+	if err != nil {
+		log.Fatalf("failed to run helm unittest: %s", out)
```

Comment:
`t.Fatalf` or `require.NoErrorf(err, "%s", out)`is preferable. `log.Fatalf` calls `os.Exit(1)` under the hood which also messes with `go test`

Reply:
Whoops I copied the code from [`chart_template_test.go`](https://github.com/redpanda-data/helm-charts/blob/160936fb09dad8dac241e7d69093bc0fc47e5335/charts/redpanda/chart_template_test.go#L39-L42) but I now see that it's a `TestMain()` function which behaves differently 🫣